packages/core/src/store.ts

```diff
@@ -41,5 +42,15 @@ export const initJsonFormsStore =
       data
     });
 
+    store.dispatch({
```

Comment:
No need to dispatch here since we can use the initial state for doing so. Note that with #780 the signature of this function will change: it will expect the overall initial state instead of multiple parameters.